mne/cov.py

```diff
@@ -161,14 +162,15 @@ def copy(self):
         """
         return cp.deepcopy(self)
 
-    def as_diag(self, copy=True):
+    def as_diag(self, copy=None):
```

Comment:
So in 0.13 this will operate in place. Maybe this should throw a warning? I think now it doesn't. Or would it make things messy?